enarx-keep-sev-shim/src/gdt.rs

```diff
@@ -15,8 +15,11 @@ use x86_64::structures::gdt::{
 use x86_64::structures::tss::TaskStateSegment;
 use x86_64::{PrivilegeLevel, VirtAddr};
 
-static mut TSS: Option<TaskStateSegment> = None;
-pub(crate) static mut GDT: Option<(GlobalDescriptorTable, Selectors)> = None;
+const STACK_NUM_PAGES: usize = 5;
+static mut STACKS: [[Page; STACK_NUM_PAGES]; 7] = [[Page::zeroed(); STACK_NUM_PAGES]; 7];
```

Comment:
```suggestion
static mut STACKS: [[Page; STACK_NUM_PAGES]; 6] = [[Page::zeroed(); STACK_NUM_PAGES]; 6];
```

?

Reply:
I don't follow how the number of stacks is chosen here

Reply:
> ?

Huh? why 6?

```rust
pub struct TaskStateSegment {
[…]
    pub interrupt_stack_table: [VirtAddr; 7],
[…]
```

Reply:
I added
```rust
debug_assert_eq!(STACKS.len(), TSS.interrupt_stack_table.len());
```